03-transactions.md

```diff
@@ -373,7 +454,7 @@ outputs) is 7140 satoshi. The final fee may be even higher if the
 
 ### Fee Payment
 
-Base commitment transaction fees are extracted from the funder's amount; if that amount is insufficient, the entire amount of the funder's output is used.
+Base commitment transaction fees and amounts for `to_local_anchor` and `to_remote_anchor` outputs are extracted from the funder's amount; if that amount is insufficient, the entire amount of the funder's output is used.
```

Comment:
This has existed since the spec was created (more or less), but this is inconsistent with an existing notice in BOLT 2 that the initiator's reserve must be up held (fees can't dig into the reserve).

Reply:
Updated